plugins/importexport/native/filter/NativeFilterHelper.inc.php

```diff
@@ -69,7 +108,7 @@ function createCoversNode($filter, $doc, $object) {
 
 				import('classes.file.PublicFileManager');
 				$publicFileManager = new PublicFileManager();
-				$filePath = $publicFileManager->getContextFilesPath($object->getData('contextId')) . '/' . $coverImage;
```

Comment:
(@NateWr is moving us away from getter functions and towards `getData` with the introduction of SchemaDAO -- so we may end up changing this back. But for now `IssueDAO` is not a `SchemaDAO`, so this is fine, just a heads-up.)

Reply:
OK